src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseFileRepository.cs

```diff
@@ -102,7 +102,8 @@ public async Task<ReleaseFile> Get(Guid releaseId, Guid fileId)
         public async Task<List<ReleaseFile>> GetByFileType(Guid releaseId, params FileType[] types)
         {
             return await _contentDbContext.ReleaseFiles
-                .Include(f => f.File)
+                .Include(rf => rf.File)
+                .ThenInclude(f => f.CreatedBy)
```

Comment:
This include for the CreatedBy user is only added here on `GetByFileType` but there are other methods returning ReleaseFiles like `Get` where it's not been added. It's not immediately obvious to the reader why `CreatedBy` is being included here, since this is a generic repository. There's nothing stopping the `ToFileInfo` extension being called on a ReleaseFile that's been retrieved using `Get` which will then have a blank email, not because it was a legacy file that didn't have a `CreatedBy` but because of a bug where we forgot to include it. It might be better to use this explicit load of `CreatedBy` like we do elsewhere for data files instead in every place before creating `FileInfo` to ensure it's always been loaded.

```
await _contentDbContext.Entry(releaseFile)
    .Reference(rf => rf.CreatedBy)
    .LoadAsync();
```